rohmu/delta/common.py

```diff
@@ -76,7 +81,7 @@ class Config:
         # possibly the tests themselves are broken
         validate_assignment = True
 
-    def jsondict(self, **kw):
+    def jsondict(self, **kw: Any) -> dict[str, Any]:
```

Comment:
`Dict`?

Reply:
There is a mix of `Dict` vs `dict` here but it's mostly due to python 3.7 support where `dict` does not support arguments.

Since I added the `from __future__ import annotations` flag in a few flag there is no issue with using `dict` with annotations.

There are a couple of places where we *must* use `Dict` (e.g. defining a type alias).

I guess I should replace all uses of `Dict` in annotations with `dict` and only leave `Dict` for type aliases until we drop python3.7 support.